save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/modal/Modal.kt

```diff
@@ -25,6 +25,7 @@ private val defaultModalStyle = Styles(
 /**
  * @param handler builder for modal component
  */
+@Deprecated("Use non-legacy code")
```

Comment:
I believe in this PR we'll need to completely remove dependency on `-legacy` artifact, if it's possible :)